etc/profile-a-l/localsend_app.profile

```diff
@@ -0,0 +1,68 @@
+# Firejail profile for localsend_app
+# Description: An open source cross-platform alternative to AirDrop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include localsend_app.local
+# Persistent global definitions
+include globals.local
+
+blacklist /usr/libexec
+
+noblacklist ${HOME}/.config/autostart
```

Comment:
```suggestion
```

This would allow trivially escaping the sandbox.

I think it would only make sense to allow this path on the profile of a program
that is dedicated to managing startup programs.

Reply:
Nice catch. Totally makes sense, thanks for the review!